Dockerfile

```diff
@@ -76,7 +66,7 @@ RUN apt update \
         yamllint \
         zip \
         unzip \
-        msodbcsql17 \
+        msodbcsql18 \
```

Comment:
@weierophinney can we update this without breaking `laminas-db` (which is not maintained anyways)?
No clue if PHP stuff is only working with v17 or if v18 is okay as well.

I was unable to get v17 installed.

Reply:
IMO OK to roll this out, regardless of `laminas-db` CI. 

Testing MsSQL in CI is painful at best anyway, from multi-year experience in doing so on `doctrine/dbal`.

Reply:
Agreed with @Ocramius — the version of msodbcsql is not terribly relevant, as they are typically BC; it just needs to be present for the extension to work. If 17 cannot be installed, move to 18.